docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Try restarting the agent. Specific instructions on how to do so depending on your environment can be found [here](https://sematext.com/docs/agents/sematext-agent/starting-stopping).
+2. Update the agent to the latest version. For more details on how to do that, see our [Agent Upgrade Instructions](https://sematext.com/docs/monitoring/spm-faq/#agent-updating).
+3. If you are experiencing high CPU usage, do the following:
+   1. Navigate to the journal directory, by default `/opt/spm/spm-monitor/st-agent/journal`.
+   2. See if it contains a large amount of files. If it does, continue with these steps.
```

Comment:
amount => number -- that is, if we are talking about number of files.  Or are we talking about volume?

Reply:
From Nedim's email, I understood number of files.

Reply:
https://writingexplained.org/amount-vs-number-difference#:~:text=Number%20is%20used%20with%20nouns,be%20individually%20counted%2C%20like%20starlight.